Source/Blazorise/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "1.0.0",
+  "name": "blazorise",
+  "private": true,
+  "devDependencies": {},
+  "dependencies": {
+    "autonumeric": "4.6.0",
```

Comment:
I'm not familiar with `dependabot`. Should this be defined with a wildcard? eg `4.6.*`?

Reply:
This is actually an NPM package.json. Dependabot will, depending on the infrastructure (nuget, npm, etc) check for updates in the right package manager.

The real "magic" for dependabot is in the .github directory. This file contains the actual versions being used by Blazorise so it knows whether you are up to date or not.

Reply:
And it knows about the major, minor, and patch versioning?

Reply:
Yes. The "only" thing you need to do is make sure the initial versions are correct in package.json and your customized files so they are not out of sync.